README.md

```diff
@@ -18,99 +18,112 @@ On an on-going basis there will be:
+$ ruby scripts/swap.rb --name abc.wgbh-mla-test.org --zone_id 123ABC
+```
 
-**asset_proxy=** V_4D37F2D8E1054BA49999027BF9D18957.mp4, V_4D37F2D8E1054BA49999027BF9D18957.webm
+Note, at the time of this documentation the script may still be called `swap_and_rsync.rb`.
 
-**asset_thumbanil=** V_4D37F2D8E1054BA49999027BF9D18957.jpg
+When that process completes, you can go to the live Open Vault URL and verify that the new code came deploy that had previously been on the demo site is now live.  You can also visit the demo url if you wish to see if the non-updated code is still in place.
 
-**asset_transcript=** V_4D37F2D8E1054BA49999027BF9D18957.xml
+Ingesting Metadata Records
+Open Vault is built around making PBCore xml records accessible.  These are generated from an Open Vault PBCore [Filemaker database](https://wiki.wgbh.org/display/MLA/Creating+and+Updating+Asset+Records).
 
-### Uploading to Amazon S3
+Exports from that database are presented as single or multiple, zipped xml documents.  If the xml is not valid PBCore, or if it's not structured in a way that will be valid for the Open Vault data modeling, it will fail to ingest those specific records and the errors are recorded in a log file.  If there are multiple xml files trying to be ingested and some are valid, they will continue to be successfully be ingested.
 
-You can use the AWS web interface to upload asset proxies, thumbnails, or transcripts for Open Vault but for uploading multiple files you should use the Amazon CLI tool.  The transfer speed is a lot faster and large transfers shouldn't time out.
+We also run the ingest process on both the demo and live server so they remain record level identical.
 
-[Follow the documentation](http://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html) to set up CLI with your Access Key, Secret Access Key, and Default region name.
+Once you have your PBCore xml export run the following commands.
+```
+$  cd /openvault3_deploy
+$ for i in `cd ../aws-wrapper && ruby scripts/ssh_opt.rb --name abc.wgbh-mla-test.org --ips_by_tag --zone_id 123ABC`; do OV_HOST=$i OV_SSH_KEY=~/.ssh/OPENVAULT_SSH_KEY_FILE bundle exec cap demo ingest OV_PBCORE=/PATH/TO/PBCORE/pbcore_xml.zip& done
+```
 
-The buckets are located under **/catalog/**
+Make sure to add the `&` after the path to your PBCore.
 
-- /asset_proxies/ is for image, audio, or video asset proxy files (.jpg, .mp3, .mp4 and .webm)
-- /asset_thumbnails/ is for asset thumbnails (.jpg)
-- /asset_transcripts/ is for asset transcript files (.xml)
+Completing that command may take some time because it's ingesting to and restarting both servers' indexes.
 
-Copy Directory of Files to S3:
+## Verify Successful Ingest
+To verify ingest completed successfully you can view the most recent ingest log files on both the demo and live servers.
 ```
-aws s3 cp /local/folder/of/stuff s3://openvault.wgbh.org/bucket-name --recursive
+$ ssh -i ~/.ssh/OPENVAULT_SSH_KEY_FILE ec2-user@12.345.678.910
```

Comment:
... and maybe the path to where the logs are? or it could be appended after this command, maybe.